frontend/pulse3d/src/components/uploadForm/AnalysisParamForm.js

```diff
@@ -140,37 +146,47 @@ const SmallLabel = styled.label`
   padding-right: 15px;
 `;
 
+const AdvAnalysisContainer = styled.div`
+display: flex;
+flex-direction row;
+height: 157px;
+`;
+
 export default function AnalysisParamForm({
-  inputVals,
```

Comment:
Why was this removed?

Reply:
analysisParams were being passed in twice